app/views/wyatt_controller/index.html.erb

```diff
@@ -1 +1,12 @@
-<h1>Hi, James!</h1>
+<!DOCTYPE html>
```

Comment:
You actually don't need to include the full HTML building blocks in each individual  view. There is a [application layout file](https://github.com/jtperreault/political_ties/blob/testing/app/views/layouts/application.html.erb) that contains all the boiler plate markup for your application. This is a pretty cool feature, if you ever need to change the primary app markup you only have to do it in 1 place, rather than scattered throughout many view files.

The other side effect is that your view files need only contain the _specific_ markup for whatever it is you're working on. 👍